_posts/2021-09-18-p2pool-released.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Monero P2Pool is now live"
+summary: P2Pool centralized pool for Monero mining is live on the mainnet

```

Comment:
```suggestion
summary: P2Pool, the decentralized pool for Monero mining, is live on mainnet

```

Reply:
Lol. That's why we review these things. Not sure how that slipped in.